src/axelrod_dojo/archetypes/hmm.py

```diff
@@ -28,11 +28,16 @@ def normalize_vector(vec):
     s = sum(vec)
     if s == 0.0:
         n = len(vec)
-        return [1 /n for v in vec]
+        return [1 / n for v in vec]
     vec = [v / s for v in vec]
     return vec
 
 def mutate_row(row, mutation_probability):
+    """
+    Given a row of probabilities, randomly change each entry with probability
+    `mutation_probability`.  Then if changing, then change by a value randomly
```

Comment:
`Then if changing, change by a value` (extra `then`)

Reply:
> A doc string would be nice, along with a comment about keeping the values within [0, 1]

Could we have:

```
Given a row of probabilities, randomly change each entry with probability
 `mutation_probability` (a value between 0 and 1).
```

Reply:
No problem.